pkg/controller/migcluster/validation.go

```diff
@@ -152,6 +158,30 @@ func (r ReconcileMigCluster) validateSaSecret(cluster *migapi.MigCluster) error
 	return nil
 }
 
+func (r ReconcileMigCluster) validateCABundle(cluster *migapi.MigCluster) {
+	if cluster.Spec.IsHostCluster {
+		return
+	}
+	if cluster.Spec.Insecure && len(cluster.Spec.CABundle) > 0 {
```

Comment:
I'm not convinced that raising a `critical` condition if the user specifies the CA bundle when it won't be used is valuable. Perhaps too strict?  Thoughts, @mansam?

Reply:
@jortel Yeah, agreed. The conditional in BuildRestConfig prevents the error, so the status is unnecessary. Fixing.